src/components/Poll/PollCreationDialog/MultipleAnswersField.tsx

```diff
@@ -67,7 +67,7 @@ export const MultipleAnswersField = () => {
                 );
               }}
               placeholder={t('Maximum number of votes (from 2 to 10)')}
-              type='number'
+              type='text'
```

Comment:
Would this cause a "full keyboard" to be shown on mobile?

Reply:
Yes, but we are not able to react to (validate) typing of "e" into the input - change event gives us empty string "". So we are not able to distinguish btw removing the text content from the inptu vs typing "e"